01_prepare_host.sh

```diff
@@ -84,8 +84,8 @@ source lib/images.sh
 
 # Add usr/local/go/bin to the PATH environment variable
 GOBINARY="/usr/local/go/bin"
-if [[ ":$PATH:" != *":$GOBINARY:"* ]]; then
-  echo export PATH="$PATH":/usr/local/go/bin >> ~/.bashrc
+if [ ":$PATH:" != *":${GOBINARY}:"* ]; then
```

Comment:
This is buggy. It requires path to be in between `:`, which it isn't if its first or last.

Also, `$PATH` is missing curlys.